Beta cmdlet docs (#881)
microsoftgraph/entra-powershell: module/docs/entra-powershell-beta/Microsoft.Graph.Entra.Beta/New-EntraBetaAdministrativeUnitMember.md

 [-AssignedLabels <System.Collections.Generic.List`1[Microsoft.Open.MSGraph.Model.AssignedLabel]>]
 [-OdataType <String>] 
 [-Description <String>] 
 -SecurityEnabled <Boolean> 
 [-IsAssignableToRole <Boolean>]
 [-ProxyAddresses <System.Collections.Generic.List`1[System.String]>] 
 -DisplayName <String>
 [-Visibility <String>] 
 -MailEnabled <Boolean> 
 -MailNickname <String> 
 [-MembershipRule <String>]
 [-MembershipRuleProcessingState <String>] 
 [<CommonParameters>]
```

## Description

The `New-EntraBetaAdministrativeUnitMember` cmdlet creates a Microsoft Entra ID object as a member of an administrative unit. Specify `Id`, `DisplayName`, `MailNickname`, `SecurityEnabled` and `MailEnabled` parameters for create a new administrative unit member.

Currently only Microsoft Entra ID groups are supported to create administrative unit members.

For information about creating dynamic groups, see Using attributes to create advanced rules (<https://azure.microsoft.com/documentation/articles/active-directory-accessmanagement-groups-with-advanced-rules/>).

## Examples

### Example 1: Create a dynamic group in an administrative unit

```powershell

> Connect-Entra -Scopes 'AdministrativeUnit.ReadWrite.All, Directory.Read.All, Directory.ReadWrite.All, Group.ReadWrite.All'

Reviewer: ```suggestion
Connect-Entra -Scopes 'AdministrativeUnit.ReadWrite.All'
```

> New-EntraBetaAdministrativeUnitMember -Id 'bbbbbbbb-1111-2222-3333-cccccccccccc' -OdataType 'Microsoft.Graph.Group' -DisplayName  'NewAUMember' -Description 'createdUnitMembe' -MailEnabled $True -MailNickname 'new' -SecurityEnabled $False -GroupTypes @('Unified','DynamicMembership') -MembershipRule "(user.department -contains 'Marketing')" -MembershipRuleProcessingState 'On' -IsAssignableToRole $false -Visibility 'Public' -ProxyAddresses {'SMTP:Ahiresh@M365x99297270.onmicrosoft.com'}

Reviewer: ```suggestion
$params = @{
    Id = 'bbbbbbbb-1111-2222-3333-cccccccccccc'
    OdataType = 'Microsoft.Graph.Group'
    DisplayName = 'NewAUMember'
    Description = 'createdUnitMember'
    MailEnabled = $True
    MailNickname = 'new'
    SecurityEnabled = $False
    GroupTypes = @('Unified', 'DynamicMembership')
    MembershipRule = "(user.department -contains 'Marketing')"
    MembershipRuleProcessingState = 'On'
    IsAssignableToRole = $false
    Visibility = 'Public'
    ProxyAddresses = @('SMTP:Ahiresh@M365x99297270.onmicrosoft.com')
}

New-EntraBetaAdministrativeUnitMember @params
```

```

### -AssignedLabels

> This parameter allows the assignment of sensitivity labels to groups. For more information on how sensitivity labels can be assigned to groups, refer to [Assign sensitivity labels](https://azure.microsoft.com/documentation/articles/active-directory-accessmanagement-groups-with-advanced-rules/)

Reviewer: ```suggestion
This parameter allows the assignment of sensitivity labels to groups. For more information on how sensitivity labels can be assigned to groups, refer to [Assign sensitivity labels](https://learn.microsoft.com/entra/identity/users/groups-dynamic-membership)
```
